packages/factory_reset_tools/lib/dbus/factory_reset.dart

```diff
@@ -3,10 +3,12 @@ import 'dart:io';
 
 import 'package:dbus/dbus.dart';
 import 'package:factory_reset_tools/dbus/dbus_remote_object.dart';
+import 'package:path/path.dart' as p;
```

Comment:
You need to define path as a dependency in the pubspec.yaml file :)

Reply:
Yes, I tried, but then other problems arose :-(

Reply:
I added it, and look what happens... :-(

Reply:
You have to update the lock file too, go into the factory_reset_tools directory and run `flutter pub get`

Reply:
```
raster ~/workspace/ubuntu-desktop-provision/packages/factory_reset_tools (accept_environment_variable_for_DESKTOP_PROVISION_path)$ flutter pub get
Resolving dependencies... 
Because factory_reset_tools depends on ubuntu_wizard any which doesn't exist (could not find package ubuntu_wizard at https://pub.dev), version solving failed.
```

Reply:
You have to run `melos bootstrap` first

Reply:
I think that I finally been able to fix it. I had a more recent flutter version, and that was breaking everything in my system.

Reply:
Ah yes, we haven't updated this repo to the latest flutter version yet. Glad you managed to solve it!

Reply:
Ok, it's ready for another review pass.